From: Arthur Zamarin <arthurzam@gmail.com>
Date: Fri, 22 Oct 2021 17:48:28 +0300
Subject: [PATCH] Fix expected error message for python 3.10

python 3.10 adds some words to it's errors, so catch if our expected
are contained in the actual output

Signed-off-by: Arthur Zamarin <arthurzam@gmail.com>
https://github.com/jaraco/cssutils/pull/17

--- a/cssutils/tests/basetest.py
+++ b/cssutils/tests/basetest.py
@@ -126,7 +126,7 @@ class BaseTestCase(unittest.TestCase):
             if not msg:
                 # No message provided: any message is fine.
                 return
-            elif excMsg == msg:
+            elif msg in excMsg:
                 # Message provided, and we got the right message: passes.
                 return
             else:
